docs/getting_started.md

```diff
@@ -144,6 +144,12 @@ Create the example guestbook application with the following command:
 argocd app create guestbook --repo https://github.com/argoproj/argocd-example-apps.git --path guestbook --dest-server https://kubernetes.default.svc --dest-namespace default
 ```
 
+Command for setting default namespace:
+
+```bash
+kubectl config set-context --current --namespace=argocd
+```
```

Comment:
Please move this section after line 139.
I would also suggest to change the text "Command for setting default namespace" to "First we need to set the current namespace to argocd running the following command:"